### ISSUE TITLE
How can i ensemble question_answering_model ?

### ISSUE THREAD

**Issue by dathudeptrai** [ThilinaRajapakse/simpletransformers#298]
Hi, Thanks for ur great implementation. I am investigating how to ensemble the results of the question answering model on many different models (such as bert, xlnet, distilbert). I tried average start_logits and end_logits but the results worse (maybe because each model have different tokenization procedure, so the start_labels and end_labels on each models are different so average cause the results worse). If we ensemble base on the final text, it's still worse because we lost the logits distribution when ensemble. Can you show me how to ensemble right? @ThilinaRajapakse

**Comment by ThilinaRajapakse** [comment]
I am not sure whether it's practical to ensemble model outputs from different models for question answering as the outputs are a span of text from the context. The average of multiple spans seems unlikely to produce a better answer than any of the spans alone from my point of view.

**Comment by dathudeptrai** [comment]
@ThilinaRajapakse, the reason why i want to ensemble is that i want to try distillation here (use ensemble output as soft-labels to teach small models),  do u think there is a way to use same tokenizer for difference models?

**Comment by ThilinaRajapakse** [comment]
Not without training all the models from scratch yourself.
